### PR TITLE
feat(multipass): add plugin for multipass

### LINKED ISSUE #9124: Multipass plugin
**Is your feature request related to a particular plugin or theme? If so, specify it.**

Not existing yet, probable new plugin "multipass".

**Is your feature request related to a problem? Please describe.**

I'd like to have autocompletion and aliases for the multipass command line tool.

**Describe the solution you'd like**

A plugin for multipass.

**Describe alternatives you've considered**

Shell aliases and functions. I am interested  in  contributing the aliases and functions.

**Additional context**
- [Multipass website](https://multipass.run/)

**Related Issues**

Nothing related found.

### PR DESCRIPTION
**Pull request by xlith**
## Standards checklist:

- [x] The PR title is descriptive.
- [x] The PR doesn't replicate another PR which is already open.
- [x] I have read the contribution guide and followed all the instructions.
- [x] The code follows the code style guide detailed in the wiki.
- [x] The code is mine or it's from somewhere with an MIT-compatible license.
- [x] The code is efficient, to the best of my ability, and does not waste computer resources.
- [x] The code is stable and I have tested it myself, to the best of my abilities.

## Changes:

- This new plugin provides completion for [multipass](https://multipass.run), as well as aliases for multipass commands

Fixes #9124

### CODE REVIEW

**Review by h4ux — COMMENTED** [review]
It's a great first version for multipass completion

**Comment by mcornella** [comment]
Can you rename the plugin to `multipass`?

File: plugins/zsh-multipass/zsh-multipass.plugin.zsh [hunk 1]
```diff
@@ -0,0 +1,43 @@
+_multipass_get_command_list () {
+    $_comp_command1 | sed "1,/Available commands/d" | awk '/^[ \t]*[a-z]+/ { print $1 }'
```
Comment (mcornella) [inline plugins/zsh-multipass/zsh-multipass.plugin.zsh:2]: Use 2-space indentation.

File: plugins/zsh-multipass/zsh-multipass.plugin.zsh [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  case $arg_name in 
+    name)
+      $_comp_command1 list | tail -n +1 | awk '/^[ \t]*[a-z]+/ { print $1 }'
+        ;;
+    command)
+      $_comp_command1 | sed "1,/Available commands/d" | awk '/^[ \t]*[a-z]+/ { print $1 }'
+      ;;
+  esac
+}
+_multipass () {
```
Comment (mcornella) [inline plugins/zsh-multipass/zsh-multipass.plugin.zsh:18]: nitpick: missing a space.

File: plugins/zsh-multipass/zsh-multipass.plugin.zsh [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+compdef _multipass multipass
+
+
+alias mp="multipass"
+alias mpla="multipass launch"
+alias mpl="multipass list"
+alias mpsp="multipass stop"
+alias mpst="multipass start"
+alias mps="multipass shell"
+alias mpln="multipass launch --network en0 --network name=bridge0,mode=manual"
```
Comment (mcornella) [inline plugins/zsh-multipass/zsh-multipass.plugin.zsh:43]: These should be alphabetically sorted.

**Review by mcornella — CHANGES_REQUESTED** [review]
Some changes needed, as well as changing the name of the plugin to `multipass`.

File: plugins/zsh-multipass/README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# zsh-multipass
+
+This plugin provides completion for [multipass](https://multipass.run), as well as aliases
+for multipass commands.
+
+To use it add `zsh-multipass` to the plugins array in your zshrc file.
+
+```zsh
+plugins=(... zsh-multipass)
+
```
[the remark this answers is not in the document]
Comment (xlith) [reply plugins/zsh-multipass/README.md]: Done!

File: plugins/zsh-multipass/zsh-multipass.plugin.zsh [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    command)
+      compadd $(_multipass_get_command_list)
+      ;;
+    *)
+      compadd $(_multipass_get_subcommand_list)
+      ;;
+  esac
+}
+
+compdef _multipass multipass
```
[the remark this answers is not in the document]
Comment (xlith) [reply plugins/zsh-multipass/zsh-multipass.plugin.zsh]: Done!

File: plugins/zsh-multipass/zsh-multipass.plugin.zsh (hunk 3 above)
Comment (xlith) [reply plugins/zsh-multipass/zsh-multipass.plugin.zsh]: Done!

File: plugins/zsh-multipass/zsh-multipass.plugin.zsh (hunk 2 above)
Comment (xlith) [reply plugins/zsh-multipass/zsh-multipass.plugin.zsh]: Done!

File: plugins/zsh-multipass/zsh-multipass.plugin.zsh (hunk 1 above)
Comment (xlith) [reply plugins/zsh-multipass/zsh-multipass.plugin.zsh]: Done!

File: plugins/zsh-multipass/zsh-multipass.plugin.zsh [hunk 6]
```diff
@@ -0,0 +1,43 @@
+_multipass_get_command_list () {
+    $_comp_command1 | sed "1,/Available commands/d" | awk '/^[ \t]*[a-z]+/ { print $1 }'
+}
+ # $_comp_command1 | sed "1,/Available commands/d" | awk '/^[ \t]*[a-z]+/ { print $1 }'
+ # $_comp_command1 list | tail -1 | awk '/^[ \t]*[a-z]+/ { print $1 }'
```
[the remark this answers is not in the document]
Comment (xlith) [reply plugins/zsh-multipass/zsh-multipass.plugin.zsh]: Done!

**Comment by xlith** [comment]
Thanks for the review. All of them should be fixed now. Please let me know if I need to do anything else.

**Review by mcornella — APPROVED** [review]

**Comment by mcornella** [comment]
Thanks Civan! And apologies for the delay!

**Pull request merged**